osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -436,12 +433,19 @@ private void joinedChannelsChanged(object sender, NotifyCollectionChangedEventAr
             {
                 case NotifyCollectionChangedAction.Add:
                     foreach (Channel channel in args.NewItems.Cast<Channel>())
-                        ChannelTabControl.AddChannel(channel);
+                    {
+                        if (channel.Type != ChannelType.Multiplayer)
```

Comment:
we're gonna want to add spectator here at some point, but i guess that's not relevant until it's implemented.

Reply:
Didn't realise we had a spectator type, can be done later. Also, what's a temporary channel?

Reply:
I believe it's an unused legacy type. Or maybe it is used but won't be relevant to lazer.